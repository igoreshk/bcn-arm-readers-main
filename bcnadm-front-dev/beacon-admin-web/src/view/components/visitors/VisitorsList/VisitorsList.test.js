import React from 'react';
import { shallow } from 'enzyme';
import { VisitorsList } from './index';

const props = {
  isLoading: false,
  location: {
    updated: false
  }
};

const wrapper = shallow(<VisitorsList {...props} />);

describe('VisitorsList', () => {
  it('Check matching snapshot', () => expect(wrapper).toMatchSnapshot());
});
