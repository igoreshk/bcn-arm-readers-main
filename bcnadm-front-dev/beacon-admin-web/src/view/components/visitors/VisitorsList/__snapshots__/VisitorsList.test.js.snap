// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VisitorsList Check matching snapshot 1`] = `
<div
  className=""
>
  <Connect(CustomAppBar)
    title={<withRouter(LocalizedComponent) />}
  />
  <ErrorBoundary
    FallbackComponent={[Function]}
  >
    <withRouter(Connect(LocalizedComponent))
      updated={false}
    />
  </ErrorBoundary>
</div>
`;
